packages/lesspass-pure/src/views/Login.vue

```diff
@@ -37,6 +37,7 @@
         v-bind:email="email"
         v-bind:showEncryptButton="true"
         v-bind:EncryptButtonText="$t('Encrypt my master password')"
+        v-bind:PlainCheckboxText="$t('Use plain password')"
```

Comment:
Master password component is used in other pages.

I'm not sure it's interesting to have this as a props.
You can use `$t('Use plain password')` directly inside the component.

or you change everywhere `<master-password/>` component (I prefer the first solution)

Reply:
Should be fixed!